crates/e2e/tests/e2e/local_node.rs

```diff
@@ -102,6 +141,18 @@ impl<T: Transport> EvmApi<T> {
         let value_id = serde_json::json!(snapshot_id);
         CallFuture::new(self.transport.execute("evm_revert", vec![value_id]))
     }
+
+    pub fn set_next_block_timestamp(&self, datetime: &DateTime<Utc>) -> CallFuture<String, T::Out> {
+        let json_timestamp = serde_json::json!(datetime.timestamp());
+        CallFuture::new(
+            self.transport
+                .execute("evm_setNextBlockTimestamp", vec![json_timestamp]),
+        )
```

Comment:
Is it required to initialize the node with an old timestamp because `evm_setNextBlockTimestamp` does not allow to set a timestamp in the past and some tests need an old block time?

Reply:
I definitely should have said it explicitly it since it's what makes everything more complicated: as you mention, time in a testing node can only be moved forward and not backward. So I initialize the node in the past to get some more time in the past when running the test. I'll update the description.